docs/DeveloperGuide.md

```diff
@@ -188,23 +153,14 @@ to only require one `Tag` object per unique tag, instead of each `Person` needin
 </div>
 ### OfficeConnectModel component
 
```

Comment:
I suggest enlarging the new umls you have included in your new PR by slightly more (maybe 1.3x ?) so that the fonts of the words can match the words in our documentation.

Also, the ### OfficeConnectModel component does not bold and enlarge for some reason :0 could it be due to the sizing of the uml diagram below? 

Lastly, I think it would be better if you can show the uml diagrams ***alone*** on a separate line by itself, instead of saying "API: <Image Here>. What I mean is to leave a breakline between the ```**API** : [`OfficeConnectModel.java`]``` and the image of the uml diagram.

This is what I see: 

![image](https://user-images.githubusercontent.com/99934242/227399151-afaae337-df1a-40d1-8f5e-3d6aaa72372b.png)

and maybe this would be better:
**API** : [`OfficeConnectModel.java`](https://github.com/AY2223S2-CS2103-F10-1/tp/blob/master/src/main/java/seedu/address/model/OfficeConnectModel.java)
<br>
<img src="images/OfficeConnectModelClassDiagram.png" width="550" />